lib/tree.rb

```diff
@@ -1,47 +1,90 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+class Tree
+  attr_reader :height, :age, :apples, :alive
+  MAX_AGE = 100
+  MAX_DIAMETER = 12
 
   def initialize
+      @height = 0.7
```

Comment:
nitpick: rubyist convention is 2 spaces. 
https://github.com/bbatsov/ruby-style-guide